daemon/albatross_console.ml

```diff
@@ -61,7 +61,8 @@ let read_console id name ring fd =
          | exn ->
            Logs.err (fun m -> m "%s error while reading %s" name (Printexc.to_string exn))
        end ;
-       Vmm_lwt.safe_close fd)
+       Vmm_lwt.safe_close fd >|= fun () ->
+       active := Vmm_core.String_map.update name update !active)
```

Comment:
and now we're at the bottom, we have a huge Lwt.catch and don't know anymore what failed.

we need to introduce one Lwt.catch for the `Lwt_unix.read` from the unikernel fd, and then for each albatross client a separate Lwt.catch to close (and update) the right fd -- but then the albatross-client fds we use Vmm_lwt.write, which should not raise any exceptions -- and thus we shouldn't need a Lwt.catch around it.

Here at the bottom, there was a read error from the unikernel fd, all we can do is close that fd. we don't need to update anything, since the albatross-clients should still get console output, e.g. when the unikernel restarts.